examples/flutter_gallery/lib/demo/material/menu_demo.dart

```diff
@@ -86,133 +86,138 @@ class MenuDemoState extends State<MenuDemo> {
+                  ),
+                  const PopupMenuItem<String>(
+                    enabled: false,
+                    child: Text('A disabled menu item'),
+                  ),
+                  PopupMenuItem<String>(
+                    value: _simpleValue3,
+                    child: const Text('Context menu item three'),
+                  ),
+                ],
+              ),
+            ),
+            // Pressing the PopupMenuButton on the right of this item shows
+            // a menu whose items have text labels and icons and a divider
+            // That separates the first three items from the last one.
+            ListTile(
+              title: const Text('An item with a sectioned menu'),
+              trailing: PopupMenuButton<String>(
+                padding: EdgeInsets.zero,
+                onSelected: showMenuSelection,
+                itemBuilder: (BuildContext context) => <PopupMenuEntry<String>>[
+                  const PopupMenuItem<String>(
+                    value: 'Preview',
+                    child: ListTile(
+                      leading: Icon(Icons.visibility),
+                      title: Text('Preview'),
+                    ),
+                  ),
+                  const PopupMenuItem<String>(
+                    value: 'Share',
+                    child: ListTile(
+                      leading: Icon(Icons.person_add),
+                      title: Text('Share'),
+                    ),
+                  ),
+                  const PopupMenuItem<String>(
+                    value: 'Get Link',
+                    child: ListTile(
+                      leading: Icon(Icons.link),
+                      title: Text('Get link'),
+                    ),
+                  ),
+                  const PopupMenuDivider(),
+                  const PopupMenuItem<String>(
+                    value: 'Remove',
+                    child: ListTile(
+                      leading: Icon(Icons.delete),
+                      title: Text('Remove'),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            // This entire list item is a PopupMenuButton. Tapping anywhere shows
+            // a menu whose current value is highlighted and aligned over the
+            // list item's center line.
+            PopupMenuButton<String>(
               padding: EdgeInsets.zero,
+              initialValue: _simpleValue,
               onSelected: showMenuSelection,
+              child: ListTile(
+                title: const Text('An item with a simple menu'),
+                subtitle: Text(_simpleValue),
+              ),
               itemBuilder: (BuildContext context) => <PopupMenuItem<String>>[
                 PopupMenuItem<String>(
                   value: _simpleValue1,
-                  child: const Text('Context menu item one'),
+                  child: Text(_simpleValue1),
```

Comment:
Strings like _simpleValue1 could be static const; presumably that would enable `const Text(_simpleValue1)`

Reply:
I did not change this part; it showed up in this PR because the indentation changed (try using `git diff -w` to see the actual change).

We can leave this for another PR.